src/utils/regex/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * hasOverlappingRegex - Checks if 2 regexes overlap each other
```

Comment:
nit. for consistency
```suggestion
 * `hasOverlappingRegex` - Checks if 2 regexes overlap each other
```